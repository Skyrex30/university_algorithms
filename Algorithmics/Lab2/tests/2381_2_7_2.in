3
1
12
Ablaki Virág
1
10
Kiss Virág
1
10
Réti Virág