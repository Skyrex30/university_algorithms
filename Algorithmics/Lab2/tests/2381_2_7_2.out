Ablaki Virág
Kiss Virág
Réti Virág
